src/php/tests/unit_tests/ChannelTest.php

```diff
@@ -81,8 +81,12 @@ public function testWatchConnectivityState()
     {
         $this->channel = new Grpc\Channel('localhost:0',
              ['credentials' => Grpc\ChannelCredentials::createInsecure()]);
-        $time = new Grpc\Timeval(1000);
```

Comment:
1000 here is 1000 microseconds, right?
I think I might get it -> we were waiting to deliver the state change from "CONNECTING" (which never happended but we acted like it did)  -> "IDLE", which should happen immediately. Because of changes in timing code in C core (we now have millisecond precision), perhaps with timeout of 1000 micros we time out immediately without delivering the status changed event.  Hence the test failure.